apps/console/src/features/applications/constants/application-management.ts

```diff
@@ -202,6 +202,8 @@ export class ApplicationManagementConstants {
     public static readonly JWT_BEARER: string = "urn:ietf:params:oauth:grant-type:jwt-bearer";
     public static readonly IWA_NTLM: string = "iwa:ntlm";
     public static readonly UMA_TICKET: string = "urn:ietf:params:oauth:grant-type:uma-ticket";
+    public static readonly DEVICE_GRANT: string = "urn:ietf:params:oauth:grant-type:device_code"
+;
```

Comment:
why is this in a new line?

Reply:
By mistake. Fixed in https://github.com/wso2/identity-apps/pull/3327/commits/dc960c94c26fe9820aaf813457713a2a3fe836ea